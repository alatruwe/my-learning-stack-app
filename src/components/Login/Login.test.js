import React from "react";
import Login from "./Login";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";

describe(`LoginForm component`, () => {
  it("renders the component by default", () => {
    const wrapper = shallow(<Login />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
