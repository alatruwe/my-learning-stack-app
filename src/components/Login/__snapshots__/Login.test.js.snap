// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login component renders the component by default 1`] = `
<ContextConsumer>
  <Component />
</ContextConsumer>
`;
